tee-worker/omni-executor/ethereum/intent-executor/src/lib.rs

```diff
@@ -43,22 +43,28 @@ impl IntentExecutor for EthereumIntentExecutor {
 		info!("Executin intent: {:?}", intent);
 		// todo: this should be retrieved from key_store
 		let signer = PrivateKeySigner::from_str(
-			"0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d",
+			"0x59c6995e998f97a5a0044964f0945389dc9e86dae86c7a8412f4603b6b78690d",
 		)
 		.unwrap();
 		let wallet = EthereumWallet::from(signer);
 		let provider = ProviderBuilder::new()
 			.with_recommended_fillers()
 			.wallet(wallet)
 			.on_http(self.rpc_url.parse().map_err(|e| error!("Could not parse rpc url: {:?}", e))?);
-		let account =
-			provider.get_account(provider.signer_addresses().next().unwrap()).await.unwrap();
+		let nonce = provider
+			.get_transaction_count(provider.signer_addresses().next().unwrap())
```

Comment:
Dumb question: what's the diff?

Reply:
`eth_getAccount` is not listend in ethereum api documentation https://ethereum.github.io/execution-apis/api-documentation/ , it worked for locally run anvil node but failed while I was testing the integration with infura's api.